src/main/java/io/jenkins/plugins/designlibrary/AppBar.java

```diff
@@ -0,0 +1,21 @@
+package io.jenkins.plugins.designlibrary;
+
+import hudson.Extension;
+
+@Extension
+public class AppBar extends UISample {
+    @Override
+    public String getDisplayName() {
+        return "App bars";
+    }
+
+    @Override
+    public String getIconFileName() {
+        return "symbol-app-bar";
```

Comment:
did you forget to add this? or you don't have a symbol yet?

Reply:
Added the icon 🙌 

<img width="115" alt="image" src="https://user-images.githubusercontent.com/43062514/160369783-b44b8a44-6879-4a61-a744-294bcd0f565d.png">